src/libs/colorpicker.c

```diff
@@ -558,7 +558,7 @@ void gui_init(dt_lib_module_t *self)
   data->color_patch = gtk_drawing_area_new();
   g_signal_connect(G_OBJECT(data->color_patch), "draw", G_CALLBACK(main_draw_callback), data);
   gtk_box_pack_start(GTK_BOX(picker_area), data->color_patch, TRUE, TRUE, 0);
-  gtk_box_pack_start(GTK_BOX(output_row), picker_area, FALSE, FALSE, 0);
+  gtk_box_pack_start(GTK_BOX(output_row), picker_area, TRUE, TRUE, 0);
```

Comment:
Cancel those changes as it's for me better on master (so without that) but with CSS tweaks and 8em instead of 7em

Reply:
I'm fine with this change. This was more personal preference.